### ISSUE TITLE
Bug with Teamtalk clacic &server

### ISSUE THREAD

**Issue by DalisonCPU** [BearWare/TeamTalk5#438]
Teamtalk Server for Windows has a very serious problem. The passwords of even not typed Channels are being viewed once you enter the cell phone.

**Comment by bear101** [comment]
If you're administrator or owner of a channel you can see its password.

**Comment by DalisonCPU** [comment]
I am not an administrator, and the password appears when I click join, the password already entered appears.
-----Mensagem original-----
De: Bjørn Damstedt Rasmussen <notifications@github.com>
Para: BearWare/TeamTalk5 <TeamTalk5@noreply.github.com>
Data: Quarta, 10 de Julho de 2019 10:41
Assunto: Re: [BearWare/TeamTalk5] Bug with Teamtalk clacic &server (#438)

If you're administrator or owner of a channel you can see its password.

**Comment by bear101** [comment]
Isn't it because you have already been in the channel?

-- Bjoern

**Comment by DalisonCPU** [comment]
Unfortunately not. They appear password of channels that I never entered, just being in Windows server.
-----Mensagem original-----
De: Bjørn Damstedt Rasmussen <notifications@github.com>
Para: BearWare/TeamTalk5 <TeamTalk5@noreply.github.com>
Data: Quarta, 10 de Julho de 2019 12:22
Assunto: Re: [BearWare/TeamTalk5] Bug with Teamtalk clacic &server (#438)

Isn't it because you have already been in the channel?



-- Bjoern

**Comment by nunotfc** [comment]
hello, let me explain:
We have our server, and created the default user with no ADM permicions.
But, when the users log in with team talk android app, i dont know if the iphone have the same problem, when they click in a channel, even if never been in it, they can see the password typed. but, in pc client version it's all normal. i think its a problem with windows server and phones client, but we use also a linux server, and all is fine.

**Comment by bear101** [comment]
I've just tested with the Windows server and an Android client. I cannot reproduce the problem.

Send a Windows server configuration where the problem occurs along with a step-by-step guide on how to reproduce the problem.

**Comment by DalisonCPU** [comment]
Hello,
this bug has already been fixed. It was because of the Portuguese translation that I was wrong.
-----Mensagem original-----
De: BjÃ¸rn Damstedt Rasmussen
Enviada em: Quinta, 01 de Agosto de 2019 22:46
Para: BearWare/TeamTalk5
Cc: DalisonCPU; Author
Assunto: Re: [BearWare/TeamTalk5] Bug with Teamtalk clacic &server (#438)

I've just tested with the Windows server and an Android client. I cannot reproduce the problem.

Send a Windows server configuration where the problem occurs along with a step-by-step guide on how to reproduce the problem.
